rocky/reports/report_types/aggregate_organisation_report/report.py

```diff
@@ -338,16 +339,16 @@ def is_mail_compliant(result):
 
         summary = {
             # _("General recommendations"): "",
-            _("Critical vulnerabilities"): total_criticals,
-            _("IPs scanned"): total_ips,
-            _("Hostnames scanned"): total_hostnames,
+            str(_("Critical vulnerabilities")): total_criticals,
```

Comment:
Would it be possible to process translation proxy objects in the JSON serializer to not have to think about stringifying these objects?

Reply:
Maybe, lets create a technical design ticket around these issues, as we'd like the json to be untranslated, where the html/pdf views would need to be translated.

Reply:
The Django encoder already does this, but processing in the JSON encoder that you can easily override is for values, not for keys. If we want to have everything multilingual we should indeed not have translated strings in the JSON and do the translation when we generate the multi organisation report.